src/vs/workbench/contrib/chat/browser/agentSessions/agentSessionsViewer.ts

```diff
@@ -243,7 +244,7 @@ export class AgentSessionRenderer implements ICompressibleTreeRenderer<IAgentSes
 			}
 
 			if (!timeLabel) {
-				timeLabel = fromNow(session.timing.endTime || session.timing.startTime, true);
+				timeLabel = fromNow(session.timing.lastRequestEnded || session.timing.lastRequestStarted || session.timing.created || 0, true);
```

Comment:
How can the time ever be `undefined` though? I would prefer we can ensure the API is sound around this and never allows `undefined`, otherwise we will play catch up game with bad behaving providers.